src/pages/gateway/fastly.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Fastly 
+description: Learn how to configure a Fastly CDN for Adobe Commerce using API Mesh.
+keywords:
+  - API Mesh
+  - Extensibility
+  - Cache
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Fastly configuration
+
+Using [Fastly](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/fastly.html?lang=en) as a content delivery network (CDN) with API Mesh for Adobe Developer App Builder provides additional security and improved performance. To fully integrate API Mesh, Fastly, and Adobe Commerce, you must follow the configuration instructions in the following sections.
+
+## Configure Fastly in Adobe Commerce
+
+Use the following process to configure edge caching for Adobe Commerce using the provided Fastly CDN.
+
+1. [Get your Fastly credentials](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/setup-fastly/fastly-configuration.html?lang=en#get-fastly-credentials).
+
+1. In the Adobe Commerce Admin, select **Store** > Settings > **Configuration** > **Advanced** > **System** > **Full Page Cache** and set the **Caching Application** field to **Fastly CDN**.
+
+1. Enter the Fastly credentials you retrieved previously into the **Fastly Service ID** and **Fastly API Token** fields. Then click the Test Credentials button to verify that your credentials are correct.
+
+1. Under **Fastly Configuration** > **Custom VCL Snippets** click **Create** and add the following [snippets](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/custom-vcl-snippets/fastly-vcl-custom-snippets.html). For more information on VCL subroutines, see [Custom Subroutines](https://developer.fastly.com/reference/vcl/subroutines/).
+
+   - Indicates when to bypass the cache:
+     - **Name** - api_mesh_bypass
+     - **Type** - **recv**
+     - **Priority** - **60**
+     - **Content**:
+
+        ```csharp
+        if ((req.request == "GET" || req.request == "HEAD") && (req.url.path~"/graphql" || req.url "^/api/(.*)") && req.url.qs~"query=" && req.http.x - commerce - bypass - fastly - cache == "true") {
+          set req.http.graphql = "1";
+        }
+        else {
+          unset req.http.graphql;
+        }
+        if (req.url.path!~"/graphql" && req.url!~"^/api/(.*)") {
+          set req.http.Magento - Original - URL = req.url;
+
+          set req.url = querystring.regfilter(req.url, "^(utm_.*|gclid|gdftrk|_ga|mc_.*|trk_.*|dm_i|_ke|sc_.*|fbclid)$");
+        }
+        ```
+
+   - Allows API Mesh to function as a backend:
+     - **Name** - api_mesh_backend
+     - **Type** - **init**
+     - **Priority** - **1**
+     - **Content**:
+  
+        ```csharp
+        backend F_graph_prod_adobe_io {
```

Comment:
we should probably put this first in the list since is the most important 
like do some sort of logic order not just provide a bulk code
the idea is to also explain each snippet a bit - what it does which we do a bit in the title:
Allows API Mesh to function as a backend
May be worth linking to Fastly backends docs?

Reply:
@cpartica to add a diagram concept